app/frontend/Calendar/DayView/EventsBlockCell.svelte

```diff
@@ -1,9 +1,7 @@
 <vbox flex class="events" on:dblclick={addEvent}>
   {#if $displayEvents && !$displayEvents.isEmpty}
     {#each $displayEvents.each as event (event.id)}
-      {#if event.startTime && event.endTime}
```

Comment:
This was a workaround for #569

Reply:
I moved the check into `WeekView` in #846 but I couldn't remove this line there due to the merge conflict.

Reply:
> I moved the check into WeekView in https://github.com/mustang-im/mustang/pull/846

I don't see the check there. Make another PR for that, if appropriate, please?